### DISCUSSION TITLE
Add an icon on the tabs to reflect whether or not the notebook is running

### DISCUSSION THREAD

**Discussion by FaintWhisper** [microsoft/vscode-jupyter#10871] [New Ideas]
Jupyter notebook tabs have an icon that provides a visual indication about the status of the notebook (running or inactive). I think this would be a nice addition for people who run multiple notebooks at the same time. This way, it wouldn't be necessary to switch tabs and check if the notebook is running or not, which is a bit annoying sometimes, especially, when you have multiple tabs open. Also, it would be nice if an icon could be added to indicate if the notebook has not run any cells.

Thank you.

**Comment by rebornix** [comment]
👍 like this idea, thanks for the suggestion.
